### PR TITLE
added a highlight option that will put a circle around the nearest dot

### PR DESCRIPTION
**Pull request by KentDi808**
screen shot 2017-09-06 at 4 28 38 pm

### CODE REVIEW

**Comment by inlineblock** [comment]
This easily could be it's own component and composed versus expanding the code of simple use scatter plot.

File: test/charts/__snapshots__/Scatterplot.test.js.snap [hunk 1]
```diff
@@ -52,6 +52,7 @@ exports[`Scatterplot.js should render the clip path 1`] = `
   }
   clipPath="my-clip-path"
   dotRadius={5}
+  highlightCircles={false}
```
Comment (mjswensen) [inline test/charts/__snapshots__/Scatterplot.test.js.snap]: Are these necessary if the default is already set to `false`?

Comment (KentDi808) [reply test/charts/__snapshots__/Scatterplot.test.js.snap]: That is the way that running "npm run jest-watch" updated them. The tests failed without them.

Comment (mjswensen) [reply test/charts/__snapshots__/Scatterplot.test.js.snap]: Oops—you're right. I was too rushed and didn't notice that these were snapshot files.

**Pull request closed without merging**